### ISSUE TITLE
Hub Events for Amplify Cognito

### ISSUE THREAD

**Issue by raphire08** [aws-amplify/amplify-flutter#659]
The sample code in the documentation mentions subscription for auth events.

https://docs.amplify.aws/lib/auth/auth-events/q/platform/flutter

What all events can be checked. I am looking for the user getting disabled from the admin side. Will hub subscription catch that, so that user will be logged out ?

**Comment by haverchuck** [comment]
@raphire08 I don't believe that is currently supported - I am marking this issue as a feature request.

**Comment by raphire08** [comment]
@haverchuck Thanks
